errors/closingError.go

```diff
@@ -0,0 +1,18 @@
+package errors
+
+import (
+	"errors"
+	"strings"
+)
+
+// IsClosingError returns true if the provided error is used whenever the node is in the closing process
+func IsClosingError(err error) bool {
+	if err == nil {
+		return false
+	}
+
+	return errors.Is(err, ErrContextClosing) ||
+		errors.Is(err, ErrDBIsClosed) ||
```

Comment:
no need for the first 2 conditions as 3rd and 4th will resolve the issues.

Reply:
done